query/src/main/scala/filodb/query/exec/MultiSchemaPartitionsExec.scala

```diff
@@ -56,13 +57,22 @@ final case class MultiSchemaPartitionsExec(queryContext: QueryContext,
     (lookupRes, Some(columnName))
   }
 
+  def isMaxMinEnabledForWorkspace(ws: Option[String]) : Boolean = {
+    ws.isDefined match {
+      case true => (GlobalConfig.workspacesDisabledForMaxMin.isDefined) &&
+        (!GlobalConfig.workspacesDisabledForMaxMin.get.contains(ws.get))
```

Comment:
This will say "max-min is disabled for this workspace" if `GlobalConfig.workspacesDisabledForMaxMin` is not defined. Is that intentional?

Reply:
Yes, that is intentional. will give the reasoning offline